index.html

```diff
@@ -249,928 +346,829 @@
-          </a>
-        </div>
+    <div class="columns">
+      <div class="column">
+        <a role="button" class="box custom-focus">
+          <div class="content">
+            <h5 class="title is-5">Front End Web Development</h5>
+            Develop Front End using latest standards with HTML5/CSS3 with added funtionality using JavaScript and
+            Vue.js.
+          </div>
+        </a>
+      </div>
+      <div class="column">
+        <a role="button" class="box custom-focus">
+          <div class="content">
+            <h5 class="title is-5">Back End Web Development</h5>
+            Develop Back End application/service using Node.js or ASP .NET and SQL server or Mongo DB databases.
+          </div>
+        </a>
       </div>
     </div>
   </div>
+</section>
 
-  <div class="section has-background-grey-lighter">
-    <div class="container">
-      <div class="section-heading has-text-centered">
-        <div class="title is-2">Photos from past events</div>
-      </div>
+<section class="section has-background-grey-lighter" aria-labelledby="photos">
+  <div class="container">
+    <div class="section-heading has-text-centered">
+      <h2 class="title is-2" id="photos">Photos from past events</h2>
+    </div>
 
-      <div class="container portfolio-container">
-        <div class="columns">
-          <div class="column is-4">
-            <div class="card">
-              <div class="card-header">
-                <p class="card-header-title">
-                  <span>Admin <small>(Light)</small></span>
-                  <span class="is-pulled-right">
-                    <a>
-                      <span class="tag is-default">v0.7.2</span>
-                    </a>
+    <div class="container portfolio-container">
+      <div class="columns">
+        <div class="column is-4">
+          <div class="card">
+            <div class="card-header">
+              <p class="card-header-title">
+                <span>Admin <small>(Light)</small></span>
+                <span class="is-pulled-right">
+                    <span class="tag is-default">v0.7.2</span>
                     <span class="tag is-default">Desktop</span>
                     <span class="tag is-danger">WIP</span>
                   </span>
-                </p>
-              </div>
-              <div class="card-content">
-                <div class="image">
-                  <img src="./img/ev-1.jpeg">
-                </div>
-              </div>
-              <div class="card-footer">
-                <a class="card-footer-item">Preview</a>
-                <a class="card-footer-item">Source Code</a>
+              </p>
+            </div>
+            <div class="card-content">
+              <div class="image">
+                <img src="./img/ev-1.jpeg" alt="JS Conf Family">
               </div>
             </div>
+            <div class="card-footer">
+              <a role="button" class="card-footer-item custom-focus">Preview</a>
```

Comment:
Not focusable by keyboard

Reply:
add `tabindex=0` for all links with the `role="button"`